kex.ini

```diff
@@ -28,6 +29,7 @@ generateAssertions = true
 logJUnit = true
 testTimeout = 10
 surroundInTryCatch = false
+maxTests = 50
```

Comment:
Also, I think it is better if this option is not enabled by default

Reply:
It has no effect if `minimizeTestSuite = false`. I just added it since its nice to know the option exists. But yes its also misleading so I'll delete it.